lib/foreman_rh_cloud/engine.rb

```diff
@@ -30,11 +30,19 @@ class Engine < ::Rails::Engine
 
         # Add permissions
         security_block :foreman_rh_cloud do
-          permission :view_foreman_rh_cloud, :'foreman_rh_cloud/reports' => [:last]
+          permission :generate_foreman_rh_cloud, :'foreman_inventory_uploads/reports' => [:generate]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/accounts' => [:index]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/reports' => [:last]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/uploads' => [:auto_upload]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/uploads' => [:download_file]
+          permission :view_foreman_rh_cloud, :'foreman_inventory_uploads/uploads' => [:last]
+          permission :view_foreman_rh_cloud, :'foreman_rh_cloud/react' => [:inventory_upload]
         end
 
         # Add a new role called 'Discovery' if it doesn't exist
-        role 'ForemanRhCloud', [:view_foreman_rh_cloud]
+        role 'ForemanRhCloud', [:view_foreman_rh_cloud, :generate_foreman_rh_cloud]
+
+        add_permissions_to_default_roles 'Organization admin' => [:view_foreman_rh_cloud, :generate_foreman_rh_cloud]
```

Comment:
I suppose we should add those permissions also to "Manager" and "System admin" roles.

Reply:
Sure, added